contracts/governance/Vesting/VestingStorage.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity ^0.5.17;
+
+import "../../openzeppelin/Ownable.sol";
+import "../../interfaces/IERC20.sol";
+import "../Staking/Staking.sol";
+import "../IFeeSharingProxy.sol";
+
+contract VestingStorage is Ownable {
```

Comment:
do you keep this `Ownable` for preserve the storage layout?
you might want to add a comment about it, otherwise it's counterintuitive

Reply:
yes, comment added